dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/model/dataRequest/RequestStatus.kt

```diff
@@ -0,0 +1,11 @@
+package org.dataland.datalandcommunitymanager.model.dataRequest
+
+import io.swagger.v3.oas.annotations.media.Schema
+
+/**
+ * An enum with the different statuses of a data request
+ */
+@Schema(
+    enumAsRef = true,
+)
+enum class RequestStatus { Open, Resolved }
```

Comment:
Just to hear your thoughts. Would it have merit to have a third status like "Pending" to indicate requests which are beeing worked on? Currently probably not, i guess.

Reply:
It is planned to introduce further statuses only when the corresponding functionality is also implemented.

Reply:
ok